src/components/common/coupon/CouponSelection.tsx

```
@@ -0,0 +1,128 @@
+            'basis-1/3 flex items-center justify-center gap-1',
+            'px-4 py-3',
+            'text-gray-30 text-sm underline',
+            'border border-gray-30 rounded-xl'
+          )}
+        >
+          <RefreshIcon />
+          최대 할인
+        </button>
+        <button
+          className={clsx(
+            'basis-2/3 flex items-center justify-center',
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)